chainer/links/activation/maxout.py

```diff
@@ -1,27 +1,46 @@
 import numpy
 
+from chainer import cuda
 from chainer.functions.activation import maxout
 from chainer import link
+from chainer.links.connection import linear
 
 
-class Maxout(link.Link):
+class Maxout(link.Chain):
     """Maxout Networks
 
+    It has three dimensional weight tensor whose shape is ``(M, C, N)``
```

Comment:
This chain actually does not hold a parameter array of that shape, so this is misleading. Make this paragraph just describe the logical formulation of this layer.